### DISCUSSION TITLE
Is it possible to load XAML runtime in Avalonia 11.x

### DISCUSSION THREAD

**Discussion by IftikharK** [AvaloniaUI/Avalonia#17062] [Q&A] [answered]
Receive XML content dynamically at runtime (e.g., through a SignalR hub) and need to create controls on the fly (Bind the dynamically created controls to the UI).
I have been searching a way to do in Avalonia, but not been successfull.

**Comment by thevortexcloud** [comment, marked as answer]
There is a runtime XAML loader if that's what you want? It will convert what ever XAML you pass into it into a .NET object.

https://github.com/AvaloniaUI/Avalonia/blob/85fff4a2320d87236c3f850b5de87129e6587458/src/Markup/Avalonia.Markup.Xaml.Loader/AvaloniaRuntimeXamlLoader.cs#L15-L25

**Reply by IftikharK** [reply]
Thanks, It means I have to install the Avalonia.Markup.Xaml.Loader Package.
